cli/util/util.go

```diff
@@ -899,3 +899,28 @@ func InstantiateFileFromTemplate(templatePath string, templateContent string,
 	}
 	return nil
 }
+
+// CollectAppList collects all the supposed applications from the passed directory,
+// considering the passed slice of exclude regexp items.
```

Comment:
Where is the slice you are talking about?

Reply:
Fixed.